### PR TITLE
fix(action button): remove focus height & width 100%

### PR DESCRIPTION
**Pull request by laraujo7**

### CODE REVIEW

**Review by GoncaloCamaz — APPROVED** [review]

**Review by nelsonmestevao — APPROVED** [review]

**Review by djcouto — APPROVED** [review]

**Review by miguelpinto98 — CHANGES_REQUESTED** [review]
This is a bypass solution. We should try to fix this behaviour correctly.

**Comment by laraujo7** [comment]
> This is a bypass solution. We should try to fix this behaviour correctly.

What I think it's happening is that the button doesn't have the props 'height: 100%' and 'width: 100%', but in the focus mode it has, so the button, when it gets focused, is trying to expand. That's why I think the solution is removing this props.

**Comment by miguelpinto98** [comment]
Try to add position absolute to the content, z-index: -1, left: 0. The before content will be the button content, and we see the blur effect working.
Right now, it's only a pixel, that push the button to down.

**Comment by laraujo7** [comment]
I added your suggestion for the focus problem. The only problem was that the left prop couldn't be 0, but it had to be the same as the padding 💪

File: packages/ui-components/src/components/action-button/action-button.scss [hunk 1]
```diff
@@ -232,9 +232,10 @@ $btn-paddings-x: (
 		&::before {
 			content: '';
 			filter: blur(2px);
-			background: $btn-focus-border-color;
-			width: 100%;
-			height: 100%;
+			position: absolute;
+			z-index: -1;
+			inset: 0;
```
Comment (miguelpinto98) [inline packages/ui-components/src/components/action-button/action-button.scss:237]: 👍

**Review by miguelpinto98 — APPROVED** [review]
let's go!

**Pull request merged**